.github/release-drafter.yml

```diff
@@ -0,0 +1,3 @@
+_extends: .github
+tag-template: ml-$NEXT_PATCH_VERSION
+version-template: $MAJOR.$MINOR.$PATCH
```

Comment:
Small nit but can you add a line feed at the end of this file

Reply:
Great!  I will add that.